### ISSUE TITLE
[All Platforms][Accessibility] Videos do not have captions

### ISSUE THREAD

**Issue by golddove** [microsoft/AdaptiveCards#4948]
# Platform

* .NET HTML
* .NET WPF
* Android
* iOS
* JavaScript
* UWP

# Version of SDK

latest main

# Details

There is no way for card authors to provide a caption/subtitle source, so renderers cannot show captions for videos. This will require adding a property to the `Media` element schema.

Seems to have been discussed slightly in #196.

**Comment by shalinijoshi19** [comment]
@matthidinger / @dclaux we are awaiting a PM proposal/spec for this issue; Assigning it to you for follow up. Thanks

**Comment by RebeccaAnne** [comment]
Duping this to the product board version of this issue.
